lsp-protocol.el

```diff
@@ -791,7 +785,15 @@ See `-let' for a description of the destructuring mechanism."
  (UnregistrationParams (:unregisterations) nil)
  (WatchKind nil nil)
  (WillSaveTextDocumentParams (:reason :textDocument) nil)
- (WorkspaceSymbolParams (:query) nil))
+ (WorkspaceSymbolParams (:query) nil)
+ ;; 3.17
+ (InlayHint (:label :position :kind) (:paddingLeft :paddingRight?))
```

Comment:
I had to rename this to `:paddingRight` (without the `?`), otherwise right padding didn't work

Reply:
Yeah, it was present in the previous implementation.